### ISSUE TITLE
Problem with temp file when using vimext

### ISSUE THREAD

**Issue by chriswl** [dunovank/jupyter-themes#61]
I get this error when trying to set a profile with the vim extension enabled:

```
$ jt -t onedork -vim
Traceback (most recent call last):
  File "/home/user/virtualenvs/ros2/bin/jt", line 11, in <module>
    sys.exit(main())
  File "/home/user/virtualenvs/ros2/local/lib/python2.7/site-packages/jupyterthemes/__init__.py", line 74, in main
    install_theme(args.theme, monofont=args.monofont, monosize=args.monosize, nbfont=args.nbfont, nbfontsize=args.nbfontsize, tcfont=args.tcfont, tcfontsize=args.tcfontsize, cellwidth=int(args.cellwidth), margins=args.margins, lineheight=int(args.lineheight), cursorwidth=args.cursorwidth, cursorcolor=args.cursorcolor, altlayout=args.altlayout, altprompt=args.altprompt, hideprompt=args.hideprompt, vimext=args.vimext, toolbar=args.toolbar, nbname=args.nbname)
  File "/home/user/virtualenvs/ros2/local/lib/python2.7/site-packages/jupyterthemes/__init__.py", line 42, in install_theme
    stylefx.remove_temp_files(vimext=vimext)
  File "/home/user/virtualenvs/ros2/local/lib/python2.7/site-packages/jupyterthemes/stylefx.py", line 69, in remove_temp_files
    os.remove(vimtemp)
OSError: [Errno 2] No such file or directory: '/home/user/virtualenvs/ros2/lib/python2.7/site-packages/jupyterthemes/vimtemp.less'
```

This is with jupyterthemes version 0.13.3 and python 2.7.12. A consequence of this is that the theme is set, but cannot be removed correctly, because some css changes remain in the `~/local/share/jupyter/nbextensions/vim_bindings.css` after a `jt -r`.

**Comment by dunovank** [comment]
Thanks for the feedback. Looks like it was caused by trying to delete the vimtemp file twice, causing a fail on the second attempt. Should be fixed in 0.13.4.

**Comment by chriswl** [comment]
This error is gone now, thanks for that! However, even in the new version, changes in the `~/local/share/jupyter/nbextensions/vim_bindings.css` file still remain after doing `jt -r`, so this file needs to be manually reset when the theme is to be reset.
